doc/changelog.asciidoc

```diff
@@ -4423,7 +4423,7 @@ Changed
     * Add `-s`/`--space` argument to `:set-cmd-text` (as `:set-cmd-text "foo "` will now set the literal text `"foo "`)
 - Ignore `;;` for splitting with some commands like `:bind`.
 - Add unbound (new) default keybindings to config. This also adds a new `<unbound>` special command.
-    * To unbind a command keybinding without binding it to a new key, you now have to bind it to `<unbound>` or it'll be readded automatically.
+    * To unbind a command keybinding without binding it to a new key, you now have to bind it to `<unbound>` or it'll be read automatically.
```

Comment:
I think this is another one that is wrong.
I'd suggest

```suggestion
    * To unbind a command keybinding without binding it to a new key, you now have to bind it to `<unbound>` or it'll be re-added automatically.
```

instead.

Reply:
Totally! That's what I get for reviewing this before my morning coffee, I thought _"how on earth did I manage to misspell 'read' this bad"_ even :rofl: